src/app/pages/AddressDetailsPage/index.tsx

```diff
@@ -31,9 +34,17 @@ export const AddressDetailsPage: FC = () => {
 
   const showDetails = showTxs || showErc20
 
+  const title = isLoading ? (
+    <Skeleton variant="text" />
+  ) : isContract ? (
+    t('contract.title')
+  ) : (
+    t('account.title')
+  )
+
   return (
     <PageLayout>
-      <SubPageCard featured title={t('account.title')}>
+      <SubPageCard featured title={title as string}>
```

Comment:
We pass here JSX.Element `<Skeleton`. Let's update SubPageCard prop title to `ReactNode`.

Reply:
Yeah, fixed that. (But it turns out we don't even need it here, since SubPageCard can handle displaying the skeleton on it's own, if we just tell it that the title is still loading, so now I'm doing that instead.